src/main/kotlin/lotto/domain/Money.kt

```diff
@@ -0,0 +1,21 @@
+package lotto.domain
+
+import lotto.constants.ErrorMessages
+
+/**
+ * 소지금을 저장하는 클래스
+ * Created by Jaesungchi on 2022.06.08..
+ */
+data class Money(var value: Int) {
```

Comment:
```suggestion
@JvmInline
value class Money(val value: Int) {
```

Money class 구현 좋습니다! 👍👍

추가 피드백을 드리자면, 현재 상황에서 `toString()`, `hashCode()`, `equals()`, `copy()`를 정의해주는 data class보다 value class를 사용하여 객체 생성 비용을 줄이게 하는 것도 나쁘지 않아보여요!

그리고 변경 가능한 value를 주는 것보다 변경이 불가능한 value를 주는게 좋아보입니다! 😊

참고
- [inline-classes](https://kotlinlang.org/docs/inline-classes.html)